internal/pkg/cli/deploy/static_site.go

```diff
@@ -20,12 +20,15 @@ import (
 	"github.com/spf13/afero"
 )
 
+type cacheUploader interface {
```

Comment:
nit: should we rename this interface?
```suggestion
type fileUploader interface {
```